tests/containers/workbenches/workbench_image_test.py

```diff
@@ -108,6 +108,18 @@ def test_ipv6_only(self, image: str, test_frame):
         finally:
             docker_utils.NotebookContainer(container).stop(timeout=0)
 
+    @pytest.mark.openshift
+    def test_image_run_on_openshift(self, image: str):
```

Comment:
currently failing on gha with

```
DEBUG:root:Pod is not ready: test-ns-notebook-tests-pod/notebook-tests-pod-865fc7988f-smh4x ({'conditions': [{'lastProbeTime': None,
 'lastTransitionTime': '2025-02-10T13:11:17Z',
 'message': '0/1 nodes are available: pod has unbound immediate '
            'PersistentVolumeClaims. preemption: 0/1 nodes are available: 1 '
            'Preemption is not helpful for scheduling.',
 'reason': 'Unschedulable',
 'status': 'False',
 'type': 'PodScheduled'}],
 'phase': 'Pending',
 'qosClass': 'BestEffort'})
```

I forgot that I can't just expect PVCs to get handled automatically on vanila Kubernetes

Reply:
this has been fixed already